tensorflow_recommenders_addons/dynamic_embedding/python/ops/shadow_embedding_ops.py

```diff
@@ -118,7 +118,7 @@ def __init__(self,
                                     dtype=self.params.key_dtype,
                                     distribute_strategy=distribute_strategy,
                                     shape=tensor_shape.TensorShape(None))
-      self._track_trackable(self.ids, ids_name, overwrite=False)
+      #self._track_trackable(self.ids, ids_name, overwrite=False)
```

Comment:
Feel free to remove it if tests and demos runs ok.

Reply:
Please delete it directly.

Reply:
amazon data is broken, https://github.com/tensorflow/datasets/pull/5047/files#diff-f1d0135661585ce9ff2864f0f316afbc1d88eb09fd3e9d7389bb63bb398f1603  and not yet merge

the other example worked.

Reply:
I think it's valid if Movieline-1m-keras pass